case-lib/lib.sh

```diff
@@ -38,10 +38,12 @@ if [ ! "$SOFCARD" ]; then
     SOFCARD=$(grep '\]: sof-[a-z]' /proc/asound/cards|awk '{print $1;}')
 fi
 
-func_lib_setup_kernel_last_line()
+func_lib_setup_kernel_last_time()
```

Comment:
As test step, we don't want check full kernel log,   
we just want check the kernel from what to current   

older code is for `/var/log/kern.log`, so it is kernel log last line
current is `journalctl`, so it is kern.log last time stamp

Reply:
So still use `kernel_last_line` will more easy to understand?

Reply:
OK, I think I understand now.

When using time, you don't have to care about lines any more (that confused me a bit). Instead you can just do this:
```shell
set_test_start_time()
{
  test_start_epoch=$(date +%s)
}

# test runs ...
journalctl --since="$test_start_epoch" $JOURNACTL_FORMAT_OPTS ...
```

Reply:
I know `date +%s`, in test process, I find catch from `journalctl -o short-iso-precise`  
for each case is more precise, because our component will write more long in 1s